7zip.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: 7zip
+  version: 22.01
+  epoch: 0
+  description: "File archiver with a high compression ratio"
+  copyright:
+    - license: LGPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: ''
+    to: mangled-package-version
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://7-zip.org/a/7z${{vars.mangled-package-version}}-src.tar.xz
+      expected-sha512: 3f391b1bd65a0654eb5b31b50f1d400f0ec38ab191d88e15849a6e4d164b7bf2ce4a6d70ec8b6e27bde1b83bb2d45b65c03129499334669e05ee025784be455a
+      strip-components: 0
+
+  - name: Configure and build
+    runs: |
+      cd CPP/7zip/Bundles/Alone2
+      mkdir -p b/g
+      # TODO: enable asm (requires jwasm or uasm)
```

Comment:
I'm curious about this. Do we know what it would require and how hard it would be? (not a blocker)